pyproject.toml

```
@@ -0,0 +1,124 @@
+[build-system]
+requires = ['setuptools', 'wheel']
+build-backend = 'setuptools.build_meta'
+
+[project]
+name = 'ctgan'
+description = 'Create tabular synthetic data using a conditional GAN'
+authors = [{ name = 'DataCebo, Inc.', email = 'info@sdv.dev' }]
+classifiers = [
+    'Development Status :: 2 - Pre-Alpha',
+    'Intended Audience :: Developers',
+    'License :: Free for non-commercial use',
+    'Natural Language :: English',
+    'Programming Language :: Python :: 3',
+    'Programming Language :: Python :: 3.8',
+    'Programming Language :: Python :: 3.9',
+    'Programming Language :: Python :: 3.10',
+    'Programming Language :: Python :: 3.11',
+    'Topic :: Scientific/Engineering :: Artificial Intelligence',
+]
+keywords = ['ctgan', 'CTGAN']
+version = '0.9.1.dev0'
+license = { text = 'BSL-1.1' }
+requires-python = '>=3.8,<3.12'
+readme = 'README.md'
+dependencies = [
+    "numpy>=1.20.0,<2;python_version<'3.10'",
+    "numpy>=1.23.3,<2;python_version>='3.10'",
+    "pandas>=1.1.3;python_version<'3.10'",
+    "pandas>=1.3.4;python_version>='3.10' and python_version<'3.11'",
+    "pandas>=1.5.0;python_version>='3.11'",
+    "scikit-learn>=1.1.3,<2;python_version>='3.10'",
```

here we should add appropriate bounds for all the python versions we support
```suggestion
    "scikit-learn>=0.24,<2;python_version<'3.10'",
    "scikit-learn>=1.1.0,<2;python_version>='3.10' and python_version<'3.11'",
    "scikit-learn>=1.1.3,<2;python_version>='3.11'",
```